pom.xml

```diff
@@ -170,6 +175,31 @@
         <artifactId>brave</artifactId>
         <version>${brave.version}</version>
       </dependency>
+
+      <dependency>
+        <groupId>com.google.api.grpc</groupId>
+        <artifactId>grpc-google-cloud-trace-v2</artifactId>
+        <version>${grpc-google-cloud-trace-v2.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-protobuf</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-stub</artifactId>
+          </exclusion>
+          <exclusion>
+            <groupId>com.google.guava</groupId>
```

Comment:
Filed https://github.com/googleapis/gapic-generator/issues/2648 ...

Reply:
Though related to it, I think it should be ok to remove `api-common` instead of this deeper one.

Reply:
revised. thanks!